homeassistant/components/kraken/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Kraken.com",
+  "options": {
```

Comment:
The main config flow is missing strings for the user form and the abort reasons are also missing.

Reply:
Done